src/controllers/NotificationController.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Tests for Notification Controllers
+ * Send HTTP requests to Notification Controllers and check mocked responses
+ */
+
+import * as express from "express";
+import { PagoPaConfig } from "../Configuration";
+import { ControllerError } from "../enums/ControllerError";
+import { HttpErrorStatusCode } from "../enums/HttpErrorStatusCode";
+import { NotificationSubscriptionRequestType } from "../enums/NotificationSubscriptionType";
+import * as NotificationAPI from "../services/NotificationAPI";
+import { NotificationSubscriptionResponseAPI } from "../types/api/NotificationSubscriptionResponseAPI";
+import { NotificationSubscriptionRequestCtrl } from "../types/controllers/NotificationSubscriptionRequestCtrl";
+import * as AppResponseConverter from "../utils/AppResponseConverter";
+import * as RestfulUtils from "../utils/RestfulUtils";
+
+export function notificationActivation(
```

Comment:
please name actions on what they do 

notificationActivation -> activateNotifications

(elsehwere the intent is not clear, `notificationActivation` sounds like a factory function for new Activations)

moreover is more aligned with `updateSubscription` (but now I'm wondering: is there a difference between activation and subscription ? if they are the same thing I'd rather chose one name between the twos)